daemon/cluster/swarm.go

```diff
@@ -430,6 +430,14 @@ func (c *Cluster) Info() types.Info {
 		info.NodeID = state.swarmNode.NodeID()
 	}
 
+	clusterInfo.WithValues(
+		info.Cluster.ID,
+		info.NodeID,
+		info.NodeAddr,
+	).Set(1)
+	clusterManagers.Set(float64(info.Managers))
+	clusterNodes.Set(float64(info.Nodes))
```

Comment:
It looks like these gauges would only be updated when an API user invokes the `/info` endpoint. I would expect them to update periodically regardless of API activity.